nodejs/sample-apps/aws-sdk/deploy/variables.tf

```diff
@@ -4,18 +4,6 @@ variable "name" {
   default     = "hello-nodejs-awssdk"
 }
 
-variable "collector_layer_arn" {
-  type        = string
-  description = "ARN for the Lambda layer containing the OpenTelemetry collector extension"
-  // TODO(anuraaga): Add default when a public layer is published.
-}
-
-variable "sdk_layer_arn" {
-  type        = string
-  description = "ARN for the Lambda layer containing the OpenTelemetry NodeJS Wrapper"
-  // TODO(anuraaga): Add default when a public layer is published.
-}
```

Comment:
Layers are region-based and var defaults cannot be computed off data sources, therefore I moved this to a local var